content/patterns/disclosure/examples/disclosure-faq.html

```diff
@@ -96,67 +92,58 @@ <h3>Parking <abbr title="Frequently Asked Questions">FAQ</abbr>s</h3>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
           <li>
-            The semantic structure of the FAQ is conveyed with native <code>dl</code>, <code>dt</code> and <code>dd</code> elements.
-            To ensure the list structure is communicated to assistive technologies, instead of applying a button role to the <code>dt</code> element, a <code>button</code> element is contained within the <code>dt</code> element.
-            Similarly, each <code>div</code> element containing answer content that can be shown or hidden by the button is a child of a <code>dd</code> element.
-            Since all the <code>dd</code> elements are present in the DOM even if some answers are hidden, the <code>dl</code> structure is always complete.
+            FAQの意味構造は、ネイティブの <code>dl</code>、<code>dt</code>および<code>dd</code> 要素を使用して伝えられます。リスト構造が支援技術に伝わるようにするため、<code>dt</code> 要素にボタンロールを適用する代わりに、<code>dt</code> 要素内に <code>button</code> 要素が含まれるようにします。同様に、ボタンによって表示または非表示にされる回答コンテンツを含む各 <code>div</code> 要素は <code>dd</code> 要素の子です。一部の回答が非表示であってもすべての <code>dd</code> 要素が DOM に存在するため、<code>dl</code> 構造は常に満たされています。
           </li>
-          <li>To help people with visual impairments identify the disclosure as interactive and make it easier to perceive that clicking either the disclosure button or its label changes the expanded state, when a pointer hovers over the button or its label, the background color changes, a border appears, and the cursor changes to a pointer.</li>
+          <li>視覚障害のある人がディスクロージャーをインタラクティブな要素として識別し、ディスクロージャーボタンまたはそのラベルをクリックすることで展開状態が変わることを認識できるように、ポインターがボタンまたはそのラベルの上にあるときに背景色が変わり、境界線が表示され、カーソルがポインターに変わります。</li>
           <li>
-            Because transparent borders are visible on some systems with operating system high contrast settings enabled, transparency cannot be used to create a visual difference between the element that is focused an other elements.
-            Instead of using transparency, the focused element has a thicker border and less padding.
-            When an element receives focus, its border changes from 0 to 2 pixels and padding is reduced by 2 pixels.
-            When an element loses focus, its border changes from 2 pixels to 0 and padding is increased by 2 pixels.
+            オペレーティングシステムのハイコントラスト設定が有効になっているシステムでは透明な境界線が表示されるため、透明を使用してフォーカスされた要素と他の要素との視覚的な違いを作り出すことはできません。透明を使用せず、フォーカスされた要素は境界線が太くなりなり、パディングが少なくなります。要素がフォーカスを受け取ると、その境界線は 0 から 2 ピクセルに変わり、パディングは 2 ピクセル減少します。要素がフォーカスを失うと、その境界線は 2 ピクセルから 0 に変わり、パディングは 2 ピクセル増加します。
```

Comment:
若干、修正してみました。いかがでしょうか？

```suggestion
            オペレーティングシステムのハイコントラスト設定が有効になっているシステムでは透明な境界線が可視になるため、透明であることを用いてフォーカスされた要素と他の要素との視覚的な違いを作り出すことはできません。透明を用いる代わりに、フォーカスされた要素は境界線が太くなり、パディングが少なくなります。要素がフォーカスを受け取ると、その境界線は 0 から 2 ピクセルに変わり、パディングは 2 ピクセル減少します。要素がフォーカスを失うと、その境界線は 2 ピクセルから 0 に変わり、パディングは 2 ピクセル増加します。
```

Reply:
スムーズになったと思います。ありがとございます！